articles/openshift/openshift-faq.md

```diff
@@ -10,196 +10,220 @@ ms.date: 11/04/2019
+### What is the maximum number of pods in an Azure Red Hat OpenShift cluster?  What is the maximum number of pods per node in Azure Red Hat OpenShift?
 
-## Is data on my cluster encrypted?
+The actual number of supported pods depends on an application’s memory, CPU, and storage requirements.
 
-By default, there is encryption at rest. The Azure Storage platform automatically encrypts your data before persisting it, and decrypts the data before retrieval. See [Azure Storage Service Encryption for data at rest](https://docs.microsoft.com/azure/storage/common/storage-service-encryption) for details.
+Azure Red Hat OpenShift 4.x has a 250 pod-per-node limit and a 100 compute node limit. This caps the maximum number of pods supported in a cluster to 250*100 = 25000.
 
-## Can I use Prometheus/Grafana to monitor my applications?
+Azure Red Hat OpenShift 3.11 has a 50 pod-per-node limit and a 20 compute node limit. This caps the maximum number of pods supported in a cluster to 50*20 = 1000.
 
-Yes, you can deploy Prometheus in your namespace and monitor applications in your namespace.
+### Can a cluster have compute nodes across multiple Azure regions?
 
-## Can I use Prometheus/Grafana to monitor metrics related to cluster health and capacity?
+No. All nodes in an Azure Red Hat OpenShift cluster must originate from the same Azure region.
 
-No, not at current time.
+### Can a cluster be deployed across multiple availability zones?
 
-## Is the Docker registry available externally so I can use tools such as Jenkins?
+Yes. This happens automatically if your cluster is deployed to an Azure region that supports availability zones. See [Availability zones](https://docs.microsoft.com/en-us/azure/availability-zones/az-overview#availability-zones) for more information.
 
-The Docker registry is available from `https://docker-registry.apps.<clustername>.<region>.azmosa.io/` 
-However, a strong storage durability guarantee is not provided. You can also use [Azure Container Registry](https://azure.microsoft.com/services/container-registry/).
+### Are control plane nodes abstracted away as they are with Azure Kubernetes Service (AKS)?
 
-## Is cross-namespace networking supported?
+No. All resources, including the cluster master nodes, run in your customer subscription. These types of resources are put in a read-only resource group.
 
-Customer and individual project admins can customize cross-namespace networking (including denying it) on a per project basis using `NetworkPolicy` objects.
+### Does the cluster reside in a customer subscription? 
 
-## Can an admin manage users and quotas?
+The Azure Managed Application lives in a locked Resource Group with the customer subscription. Customers can view objects in that RG but not modify them.
 
-Yes. An Azure Red Hat OpenShift administrator can manage users and quotas in addition to accessing all user created projects.
+### Is there any element in ARO shared with other customers? Or is everything independent?
```

Comment:
No acronyms. This should be Azure Red Hat OpenShift :)

Reply:
Thought I got all those! Good catch.